### PR TITLE
add Autodesk FBX

### PR DESCRIPTION
**Pull request by diedyesterdaywashere**
### SteamDB app page links to a few games using this

https://steamdb.info/app/49520/depots/

### Brief explanation of the change

added a rule for Autodesk FBX SDK

### CODE REVIEW

File: rules.ini [hunk 1]
```diff
@@ -183,6 +183,7 @@ AMD_FSR2 = (?:^|/)ffx_fsr2_api_x(?:64|86)\.dll$
 AMD_GPU_Services = (?:^|/)amd_ags_x(?:64|86)\.dll$
 AMD_TrueAudio_Next = (?:^|/)TrueAudioNext\.dll$
 Anzu = (?:^|/)anzu\.dll$
+Autodesk_FBX = (?:|/)fbxsdk
```
Comment (Lifeismana) [inline rules.ini:186]: to also match the unity plugin
```suggestion
Autodesk_FBX = fbxsdk
```

File: tests/types/SDK.Autodesk_FBX.txt
Comment (Lifeismana) [inline tests/types/SDK.Autodesk_FBX.txt:1]: add this `Subnautica_Data/Plugins/x86_64/UnityFbxSdkNative.dll`
and run the generate strings script

**Review by Lifeismana — CHANGES_REQUESTED** [review]
also needs some non matching tests

**Comment by diedyesterdaywashere** [comment]
image

fixed it

**Pull request merged**